src/ttauri/version.hpp.in

```diff
@@ -0,0 +1,41 @@
+// Copyright Take Vos 2020.
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt or copy at https://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <string>
+
+namespace tt {
+
+struct version {
```

Comment:
Can we name the type git_version? I am afraid that I would need other versions inside ttauri.

Reply:
In my thinking `tt:version` is the struct holding the versioning meta data for the library ttauri.
you want it to be a general meta data holder? why?